mlrun/model_monitoring/db/tsdb/tdengine/tdengine_connector.py

```diff
@@ -544,11 +545,7 @@ def get_last_request(
         )
         df[mm_schemas.EventFieldType.LAST_REQUEST] = df[
             mm_schemas.EventFieldType.LAST_REQUEST
-        ].map(
-            lambda last_request: datetime.strptime(
-                last_request, "%Y-%m-%d %H:%M:%S.%f %z"
-            ).astimezone(tz=timezone.utc)
-        )
+        ].map(lambda last_request: parser.parse(last_request).astimezone(timezone.utc))
```

Comment:
* Can we use a dataframe operation for this?
* Maybe we can add a test for it in `test_tdengine_connector.py`?